src/components/Form/UploadDocumentForm.jsx

```diff
@@ -84,47 +96,99 @@ const UploadDocumentForm = () => {
       statusType="Upload status"
       defaultMessage="To be uploaded..."
     >
-      <form onSubmit={handleDocumentUpload} autoComplete="off">
-        <label htmlFor="verify-checkbox">
-          Verify File on upload:
-          <input
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleDocumentUpload} autoComplete="off">
+          <FormControlLabel
+            control={<Checkbox />}
+            label="Verify file on upload"
             id="verify-checkbox"
-            type="checkbox"
             value={state.verifyFile}
-            onClick={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
+            onChange={() => dispatch({ type: 'TOGGLE_VERIFY_FILE' })}
           />
-        </label>
-        <div style={formRowStyle}>
           <DocumentSelection
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
```

Comment:
Think we can remove these Lines for 115 to 116 without impacting functionality.

That means we can also remove the `id` attribute in Line 119.